lms/djangoapps/ora_staff_grader/views.py

```diff
@@ -44,31 +44,31 @@ class InitializeView(RetrieveAPIView):
     permission_classes = (IsAuthenticated,)
 
     def get(self, request, *args, **kwargs):
-        ora_location = request.query_params['ora_location']
+        ora_location = request.query_params.get('ora_location')
+
+        if not ora_location:
+            return HttpResponseBadRequest(_("Query must contain an ora_location param."))
```

Comment:
Isn't this kind of redundant to the exception on line 58?

Reply:
No, this is the case where the `ora_location` is missing entirely. Line 58 is when an `ora_location` is provided but is either not found or is not a valid key.